### PR TITLE
Created prerequisites.md

### PR DESCRIPTION
**Pull request by AmieDD**
thundernetes prerequisites document.

### CODE REVIEW

**Comment by AmieDD** [comment]
@khaines and @dgkanatsios can you look over and see if there is anything I'm missing? We can continue to improve and add to this document as we progress with [thundernetes](https://github.com/PlayFab/thundernetes).

File: docs/prerequisites.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Prerequisites
```
Comment (dgkanatsios) [inline docs/prerequisites.md:1]: should we link to this document from README.md in the root folder?

File: docs/prerequisites.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+### Kubernetes Resources 
+- [Kubernetes Basics](https://kubernetes.io/docs/tutorials/kubernetes-basics/)
+- [Kubernetes core concepts for Azure Kubernetes Service (AKS)](https://docs.microsoft.com/en-us/azure/aks/concepts-clusters-workloads)
+- [kubectl](https://github.com/virtual-kubelet/virtual-kubelet)
+- [kind](https://kind.sigs.k8s.io/)
+  -   kind is a tool for running local Kubernetes clusters using Docker container “nodes”.
+kind was primarily designed for testing Kubernetes itself, but may be used for local development or CI.
+
+## Game Servers 👾
+Thundernetes is an preview project from the Azure PlayFab Multiplayer Servers (MPS) team that enables you to run Linux game servers that use the PlayFab Game Server SDK (GSDK) on your Kubernetes cluster.
```
Comment (dgkanatsios) [inline docs/prerequisites.md:25]: nit: "a" instead of "an"

Comment (dgkanatsios) [inline docs/prerequisites.md:25]: now that the project is expanding, should we call it "teams from Azure and XBOX"? :)

File: docs/prerequisites.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+- [kubectl](https://github.com/virtual-kubelet/virtual-kubelet)
+- [kind](https://kind.sigs.k8s.io/)
+  -   kind is a tool for running local Kubernetes clusters using Docker container “nodes”.
+kind was primarily designed for testing Kubernetes itself, but may be used for local development or CI.
+
+## Game Servers 👾
+Thundernetes is an preview project from the Azure PlayFab Multiplayer Servers (MPS) team that enables you to run Linux game servers that use the PlayFab Game Server SDK (GSDK) on your Kubernetes cluster. 
+
+- [Integrating Game Servers with Game Server SDK (GSDK)](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/integrating-game-servers-with-gsdk)
+- [Azure PlayFab Multiplayer Servers Gameserver SDK (GSDK)](https://github.com/PlayFab/MpsSamples)
```
Comment (dgkanatsios) [inline docs/prerequisites.md:28]: that's the GSDK samples repo, we can keep it. The actual GSDK repo is [https://github.com/PlayFab/gsdk](https://github.com/PlayFab/gsdk)

**Review by dgkanatsios — APPROVED** [review]
:shipit:

**Comment by dgkanatsios** [comment]
@AmieDD some minor comments, otherwise LGTM! Great work!

File: docs/prerequisites.md (hunk 1 above)
Comment (AmieDD) [reply docs/prerequisites.md:1]: Yes. Good idea!

**Review by AmieDD — COMMENTED** [review]
Review changes for prerequisites document resolved.

**Pull request merged**